UnicornEngine/video/include/unicorn/video/Renderer.hpp

```diff
@@ -43,8 +44,8 @@ class Renderer
     virtual bool Init() = 0;
     virtual void Deinit() = 0;
     virtual bool Render() = 0;
-    UNICORN_EXPORT std::shared_ptr<Camera> GetCamera() const;
-    virtual std::shared_ptr<geometry::Mesh> SpawnMesh() = 0;
+    UNICORN_EXPORT Camera& GetCamera() const;
+    virtual geometry::Mesh& SpawnMesh() = 0;
```

Comment:
Where is `DeleteMesh(Mesh*)` that deconstructs the mesh?
And since `DeleteMesh` works on pointer, `SpawnMesh` shall return a pointer too.